inc/admin/class-envato-market-admin.php

```diff
@@ -960,8 +963,11 @@ function ( $val ) {
 		 * @since 1.0.0
 		 */
 		public function authorize_token_permissions() {
+			if (wp_get_environment_type() === 'development') {
+				return 'success';
+			}
+
 			$response = envato_market()->api()->request( 'https://api.envato.com/whoami' );
-			$notice   = 'success';
```

Comment:
We may need to bring this back otherwise the default return value below would be `null` instead of `'success'`

Reply:
oh yeah whoops 👁️

Reply:
done 4c5e77b030d6f4d0fc8a39a087a0bcc3212acd10